src/index.js

```diff
@@ -0,0 +1,175 @@
+//application states
+const state = {
+  sky: 'sunny',
+  temperature: 0,
+  city: '',
+};
+const temperatureChangeStep = 1;
+
+//references to all elements on the page
+const elTemperature = document.querySelector('#temperature');
+const elIncreaseTemp = document.querySelector('#increaseTemp');
+const elDecreaseTemp = document.querySelector('#decreaseTemp');
+const elLandscape = document.querySelector('#landscape');
+const elCity = document.querySelector('#city');
+const elCityInput = document.querySelector('#cityInput');
+const elCityReset = document.querySelector('#cityReset');
+const elCitySearch = document.querySelector('#citySearch');
+const elSky = document.querySelector('#sky');
+const elSkyInput = document.querySelector('#skyInput');
+
+//event handlers to intercept user interactions
+elSkyInput.addEventListener('change', (event) => {
+  setSky(event.target.value);
+});
+
+elCityInput.addEventListener('input', (event) => {
+  setCity(event.target.value);
+});
+
+elCityReset.addEventListener('click', (event) => {
+  setCity('');
+  elCityInput.focus();
+});
```

Comment:
There's an issue with wave 03 (naming the city), the requirements specify that there should be an element that displays a city name & a text input element that allows the user to change the city name. The city name must also be updated every time there's a change to the text input element. Currently your code is not displaying the city name when you reset the city.

Reply:
When you reset the city it should be updated to a selected city instead of an empty string like so:

  cityNameInput.value = 'Seattle';